src/runner/browser-env/vite/socket.ts

```diff
@@ -103,4 +103,21 @@ function handleBrowserEvents(
 
         io.to(runUuid).except(socket.id).emit(BrowserEventNames.initialize, payload);
     });
+
+    socket.on(BrowserEventNames.runBrowserCommand, async (payload, cb) => {
+        const { runUuid } = socket.handshake.auth;
+        socket.join(runUuid);
```

Comment:
TODO: there is no need to add socket in room on this event, so just remove this line.